docs/3.0.0-beta.x/plugin-development/frontend-development.md

```diff
@@ -291,3 +291,12 @@ class Foo extends React.Component {
   }
 }
 ```
+### Backend requests
+When making requests as admin to plugin routes the authorization token needs to be included. The easiest way to accomplish this is to use _request_ method from _strapi-helper-plugin_ package.
+
+```js
+import { request } from 'strapi-helper-plugin';
+...
+const result = await request(url, { method: "GET" });
```

Comment:
Maybe you can add some documentation about the `url`. If the `requestURL` starts with a `/`  the utility automatically prefixes the endpoint with the `backendURL`  according to the current environment.

Reply:
Also, other arguments can be provided to the utility they are defined in the following order:

1. `url` | `string` | endpoint
2. `options` | `object` | `default={}` | you can provide custom headers in it
3. `shouldWatchServerRestart` | `boolean` | `default=false`| this option is mainly used when you want to know when the server has restarted. It pings the `_health` endpoint and locks the app until the server is running back
4. `stringify` | `bool` | `default=true` | stringifies the body of the request
5. `noAuth` | `bool`| `default=true` | prevent the utility from adding the `Authorization` header in the request it is useful when you want to fetch an external API.

Also, the next major release will add `axios` in the administration panel in order to have another http client that user will be able to use but we will take care of providing the documentation about it.